.github/workflows/lint.yml

```diff
@@ -80,7 +80,6 @@ jobs:
           script: |
             return [...new Set(${{ steps.filter.outputs.modules_files }}
               .map(path => path
-                .replace('tests/', '')
```

Comment:
what was the issue there?

Reply:
The issue was that when a modification was made to the `tests/` folder the succeeding splice operation wasn't performing well.
I've added this modification in another [PR](https://github.com/nf-core/modules/pull/7079/files) but it was rendered obsolete by a new PR.